BookingProject/BookingWebAPI/BookingWebApi.Application/User/Services/AppUserService.cs

```diff
@@ -39,6 +40,23 @@ public async Task<Result<UserInfo>> GetUserInfoAsync(string userId)
             return Result<UserInfo>.Success(userInfo.Value);
         }
 
+        public async Task<string> GetUserRoleByIdAsync(string userId)
+        {
+            if(userId == null)
+            {
+                return "UserId is null";
```

Comment:
??? How do u validate it then?
return Result and keep consistent with the chosen approach of this project